packages/itinerary-body/src/styled.tsx

```diff
@@ -366,11 +366,7 @@ export const LineColumn = styled.div`
   padding-right: 5px;
 `;
 
-export const LegDetails = styled.span`
-  *:not(.fa) {
-    vertical-align: middle;
-  }
-`;
+export const LegDetails = styled.span``;
```

Comment:
I can't find the answer to this anywhere, but is it alright to leave this styled component blank?

Reply:
It's fine to leave it. Actually, it is an exported symbol, and implementers could apply styles to that symbol by writing `styled(LegDetails)`.

Reply:
Gotcha, that makes sense! Good to know for the future